src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,879 @@
+                await testClient.AuthorizationServers.ActivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+
+                var retrievedDeactivatedAuthorizationServer = await testClient.AuthorizationServers.GetAuthorizationServerAsync(createdAuthorizationServer.Id);
+                retrievedDeactivatedAuthorizationServer.Status.Should().Be("ACTIVE");
+            }
+            finally
+            {
+                await createdAuthorizationServer.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationServer.Id);
+            }
+        }
+
+        [Fact(Skip = "OKTA-303121: Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task ListAuthorizationServerPolicies()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:Test AuthZ Server ({TestClient.RandomString(4)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testPolicy = new Policy
+            {
+                Type = PolicyType.OauthAuthorizationPolicy,
+                Status = "ACTIVE",
+                Name = "Test Policy",
+                Description = "Test policy",
+                Priority = 1,
+                Conditions = new PolicyRuleConditions
+                {
+                    Clients = new ClientPolicyCondition
+                    {
+                        Include = new List<string> { "ALL_CLIENTS" },
+                    },
+                },
+            };
+
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testPolicy, createdAuthorizationServer.Id);
+            try
+            {
+                var policies = await testClient.AuthorizationServers.ListAuthorizationServerPolicies(createdPolicy.Id).ToListAsync();
+                policies.Should().NotBeNull();
+                policies.Count.Should().BeGreaterThan(0);
+                policies.Select(p => p.Id).ToHashSet().Should().Contain(createdPolicy.Id);
+            }
+            finally
+            {
+                await createdPolicy.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdPolicy.Id, createdPolicy.Id);
```

Comment:
Did you mean to pass the policy id twice here?

Reply:
oops! No.

Reply:
Let's double-check we have an alias for this, we should be able to do the following:
```
await createdAuthorizationServer.DeletePolicyAsync(createdPolicy.Id);
```